source/reference/api/start.txt

```diff
@@ -111,6 +120,61 @@ Request Body Parameters
 
        Default value is ``false``.
 
+
+.. _c2c-api-start-sharding:
+
+Sharding Parameters
+~~~~~~~~~~~~~~~~~~~
+
+Synchronization from a replica set to a sharded cluster requires the 
+``sharding`` option to shard collections on the destination cluster.
+
+The ``sharding`` option has the following parameters:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 15 15 70
+
+   * - Parameter
+     - Type
+     - Description
+
+   * - ``createSupportedIndexes``
+     - boolean
+     - Sets whether synchronization creates a supporting index 
+       for the shard key, if none exists.
+
+   * - ``shardingEntries``
+     - array
```

Comment:
[question]

Would it be helpful to specify that this is an array of objects?

Reply:
Fixed